CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ This project attempts to follow [semantic versioning](https://semver.org/).
     * No more group_vars directory
     * All of the host/group configuration is in that one file now!
   * BREAKING: `subspace vars` is now `subspace secrets`
+  * BREAKING: sidekiq_concurrency renamed to sidekiq_workers, default changed from 10 -> 1
```

Comment:
can you get rid of the default instead?

Reply:
I left it at one which is safe but I see your point